README.md

```diff
@@ -27,6 +27,31 @@ $client = new StoryblokClient(
 $client->request('GET', '/api/something', $options);
 ```
 
+## Spaces
+
+In your code you should type-hint to `SensioLabs\Storyblok\Api\SpacesApiInterface`
+
+### Get the current space
+
+Returns the space associated with the current token.
+
+```php
+use SensioLabs\Storyblok\Api\SpacesApi;
+use SensioLabs\Storyblok\Api\StoryblokClient;
+
+$client = new StoryblokClient(/* ... */);
+$spacesApi = new SpacesApi($client);
+
+$response = $spacesApi->me();
+
+// Example response
+$space = $response->getSpace();
+$space->getName(); // Example Space
+$space->getDomain(); // https:/example.com
```

Comment:
```suggestion
$space->getDomain(); // https://example.com
```

Reply:
There is no getter anyway. docs are oudated